bower.json

```diff
@@ -18,12 +18,7 @@
     "js"
   ],
   "dependencies": {
-    "purescript-control": "*",
```

Comment:
I'd like to keep the explicit dependencies. I'm more than happy to accept this PR and make the changes afterward. So, there's nothing you need to do in this PR. If you feel up to adding the dependencies that's fine as well. I just wanted to let you know up front that they will be added back, so it didn't seem passive aggressive.